### DISCUSSION TITLE
Are there any plans to create a paged.js-based component?

### DISCUSSION THREAD

**Discussion by enchic** [vuejs/vitepress#4069] [Q&A]
hello. I plan to use VitePress to print PDFs or create ePubs.

I found the vitepress-export-pdf plugin to be very useful for PDF output.

And, I recently found [Paged.js](https://github.com/pagedjs/pagedjs). First, I plan to use it to use custom pagination for output in a custom template, but I'm not sure if it will work as I want.

I am wondering if there are any plans to create a paged.js-based component that can be very useful even when creating multiple markdown files into one PDF document.

Thank you for the great service.

**Comment by ritchiemarkov** [comment]
Good one. But I think we have to reach paged js authors. It's more related to them even if they are pretty framework agnostic. I have demanded em few times but had sadly no response. Idk what's going on with them. Maybe more focused on their internal projects. And yes paged is pretty cool btw but lacking of support because there is just max 2 members helping you out. I'm with you broths but we have to ask em.
